src/TestPEProjects/SizeBenchV2.AnalysisEngine.Tests.Rust/.gitignore

```diff
@@ -0,0 +1 @@
+target/*
```

Comment:
I think it should be just `target/`, with no asterisk.  Asterisk matches files.  `/` at the end matches the entire directory tree.

Reply:
Thanks, will fix